wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/wsplugins/PluginMetaRetriever.java

```diff
@@ -55,19 +56,24 @@
   private static final Logger LOG = LoggerFactory.getLogger(PluginMetaRetriever.class);
   private static final String CHE_PLUGIN_OBJECT_ERROR =
       "Che plugin '%s:%s' configuration is invalid. %s";
-  private static final String PLUGIN_REGISTRY_PROPERTY = "che.workspace.plugin_registry_url";
+  private static final String CHE_REGISTRY_MISSING_ERROR =
+      String.format(
+          "Workspace requested ws.next plugins/editor but plugin registry is not configured. "
```

Comment:
As far as I remember there was a discussion that usage of workspace next is not good. Maybe Che Editor/Plugins would be informative enough?

Reply:
yeah, you are completely right!

Reply:
Good point; fixed.